scala/akkahttp/build.sbt

```diff
@@ -14,6 +14,7 @@ assemblyOption in assembly := (assemblyOption in assembly).value
 assemblyJarName in assembly := "server_scala_akkahttp"
 
 libraryDependencies ++= Seq(
-  "com.typesafe.akka" %% "akka-http" % "[10.0,10.1)",
```

Comment:
could we use `[10.1,10.2)` in order to stick to **10.1** but have last up-to-date patch version ?

Reply:
I don't understand what you mean here. Akka Http latest version is 10.1.11. Before that, 10.1.10.

Reply:
I mean that using **10.1.11** specify that we want to use a particular version.
Using `[10.1,10.2)` range will use that last version in **10.1** (as I understand)

Reply:
I'd rather have a fixed version so that with every build you're sure you're testing the same thing. But it's your call. you could put `[10.1,10.2)` if you want... or better `[10.1.11,10.2)`... it should work okay.

Reply:
the thing is :
+ I have to stick to the version displayed in results (for convenience reason I display `X.Y`)
+ I want the less maintenance works
in consequence of what, I prefer `[10.1.11,10.2)` (this is also the case for `java` and `kotlin`)

Reply:
Ok